examples/generate.sh

```diff
@@ -103,20 +103,20 @@ kustomize build "${SOURCE_DIR}/machinedeployment" | envsubst >> "${MACHINEDEPLOY
 echo "Generated ${MACHINEDEPLOYMENT_GENERATED_FILE}"
 
 # Generate Cluster API provider components file.
-CAPI_BRANCH=${CAPI_BRANCH:-"v0.2.3"}
+CAPI_BRANCH=${CAPI_BRANCH:-"v0.2.5"}
```

Comment:
Shouldn't this default to "stable"?

Reply:
ack. will revise some of the version stuff in a later PR